platform/bcm28xx/rules.mk

```diff
@@ -42,8 +42,8 @@ SMP_CPU_ID_BITS := 8
 GLOBAL_DEFINES += \
 	BCM2836=1
 
-MODULE_SRCS += \
-	$(LOCAL_DIR)/uart.c
+MODULES += \
```

Comment:
Use MODULE_DEPS instead of MODULES. MODULES sometimes works but the _DEPS is the correct solution.